common/src/main/java/com/wynntils/features/user/ChatTabsFeature.java

```diff
@@ -29,19 +30,20 @@
 import net.minecraft.client.gui.screens.ChatScreen;
 import net.minecraftforge.eventbus.api.EventPriority;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
+import org.lwjgl.glfw.GLFW;
 
 public class ChatTabsFeature extends UserFeature {
     public static ChatTabsFeature INSTANCE;
 
     @Config(visible = false)
     public List<ChatTab> chatTabs = Arrays.asList(
-            new ChatTab("All", false, null, null),
-            new ChatTab("Global", false, Sets.newHashSet(RecipientType.GLOBAL), null),
-            new ChatTab("Local", false, Sets.newHashSet(RecipientType.LOCAL), null),
-            new ChatTab("Guild", false, Sets.newHashSet(RecipientType.GUILD), null),
-            new ChatTab("Party", false, Sets.newHashSet(RecipientType.PARTY), null),
-            new ChatTab("Private", false, Sets.newHashSet(RecipientType.PRIVATE), null),
-            new ChatTab("Shout", false, Sets.newHashSet(RecipientType.SHOUT), null));
+            new ChatTab("All", false, null, null, null),
+            new ChatTab("Global", false, null, Sets.newHashSet(RecipientType.GLOBAL), null),
+            new ChatTab("Local", false, null, Sets.newHashSet(RecipientType.LOCAL), null),
+            new ChatTab("Guild", false, "/g ", Sets.newHashSet(RecipientType.GUILD), null),
+            new ChatTab("Party", false, "/p ", Sets.newHashSet(RecipientType.PARTY), null),
+            new ChatTab("Private", false, "/msg ", Sets.newHashSet(RecipientType.PRIVATE), null),
+            new ChatTab("Shout", false, null, Sets.newHashSet(RecipientType.SHOUT), null));
```

Comment:
I've never done a shout, but is there not a command for that?

Reply:
there is not a command for shouts

Reply:
Ok. Out of curiousity, how to you do shouts? From a web page?

Reply:
/use -> bombs -> shout bomb (buy it from store)